README.md

```diff
@@ -1,51 +1,62 @@
 # Conflux-Rust
-[![Travis](https://travis-ci.com/Conflux-Chain/conflux-rust.svg?branch=master)](https://travis-ci.com/Conflux-Chain/conflux-rust#)
+<!--div align="center">
+  <img src="logo">
+</div-->
+----------------
 
-Conflux-rust is a rust-based implementation of Conflux protocol, it is fast and reliable.
+|**`Travis`**|
+|---------------|
+|[![Travis](https://travis-ci.com/Conflux-Chain/conflux-rust.svg?branch=master)](https://travis-ci.com/Conflux-Chain/conflux-rust#)|
 
-# Build Instruction
+**Conflux-rust** is a rust-based implementation of Conflux protocol, it is fast and reliable.
 
-1. Install rust.
-2. rustup update.
-3. Make sure you are using the stable branch.
-4. Install clang for compiling rocksdb.
-5. Install other dependencies for rocksdb. Instruction can be found in https://github.com/facebook/rocksdb/blob/master/INSTALL.md
+## Build Instruction
 
-# Run Instruction
+1. Install Rust. Instructions can be found [here](https://www.rust-lang.org/).
+2. Run `rustup update` to make sure that you are on the updated and stable branch
+3. Install clang for compiling rocksdb.
+4. Install other dependencies for rocksdb. Instructions can be found [here](https://github.com/facebook/rocksdb/blob/master/INSTALL.md).
+5. To build the project:
+```
+cargo build
+```
 
-1. Build the project.
-2. Enter directory `run`.
-2. Edit `default.toml`: 
+## Run Instruction
+
+1. Edit the configuration file `run/default.toml`: 
     * Set `public_address` according to your public IP.
-    * Set `mining_author` to the account address to receive mining reward.
-3. Execute the command
-        `./../target/release/conflux --config default.toml`
+    * If `start_mining=true`:
```

Comment:
"If you want to start mining, ... . Otherwise set `start_mining=false`" more clear.